server/sandbox_update_resources.go

```diff
@@ -1,15 +0,0 @@
-package server
```

Comment:
hm I actually would prefer we didn't do this change so it continues to match the container variant